keyboards/handwired/lovelive9/lovelive9.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "quantum.h"
+#ifdef RGBLIGHT_ENABLE
```

Comment:
This isn't actually needed, as it's included because you've included `quantum.h`

However, it may be worth adding the extern for the rgblight config structure here, so that it's automatically included in the keymap.c file.